### DISCUSSION TITLE
Unibox.obj  boundary defind

### DISCUSSION THREAD

**Discussion by BoyuanTang331** [InteractiveComputerGraphics/SPlisHSPlasH#292] [Q&A]
Hi,

I have tried several scenes for fluid simulation, I have a question about the boundary of the scene, 

like in the custom_scene.py file, when the particle moves during the simulation, I see all the particles reach the boundary wall x∈(-2,2) y∈(-2.2), but when I change the Camera_Lookat to observe those particles, I found they didn't really reach the boundary, is that because the Unibox.obj define a boundary which has a thickness? 

I need to get a certain distance between the particle and the boundary Wall for my later program,  can I directly set the wall position as the Unibox. obj that a cube in x∈(-2,2) y∈(-2.2) z∈(0.4)? or should I consider the thickness of the wall when particles collision to the boundary?

**Comment by janbender** [comment]
There is a small gap between the boundary and the fluid. The reason is that the boundary forces are a bit too strong and push the particles away. However, this is a problem of all boundary handling methods and as far as I know there exists no perfect solution yet.

**Comment by janbender** [comment]
If you add additional thickness, the gap will get larger. You can try to add some negative thickness.
